scripts/zap_result_scanner/lib/zap_result.rb

```diff
@@ -0,0 +1,50 @@
+require 'json'
+
+module ZAPResult
+  Result = Struct.new(:name, :confidence, :risk, :url, :param, :evidence, :alert)
+  class << self
+
+    def project_results(proj, path)
+      json = read_json(project_path(proj, path))
+      return json_to_results(proj, json)
+    end
+
+    def json_files_from_dir(path)
+      return Dir["#{path}/*.json"]
+    end
+
+    def read_json(path)
+      return File.exists?(path) ? JSON.parse(File.read(path)) : []
+    end
```

Comment:
it's meant to return an actual JSON obj, not a string.   According to the Ask missing files could be common, in the script it's useful to think of a missing results file as producing an empty Result list ([]).  Maybe the method name could be more clear?